common/scripts/docker/installDb.sh

```diff
@@ -5,28 +5,7 @@ export TIMEOUT=180
 
 __check_db() {
   __process_msg "Checking database container status on: $DB_IP:$DB_PORT"
-  local interval=3
-  local counter=0
-  local db_booted=false
-
-
-  __process_msg "Waiting 10 seconds for db boot"
-  sleep 10
-
-  while [ $db_booted != true ] && [ $counter -lt $TIMEOUT ]; do
-    if nc -vz $DB_IP $DB_PORT &>/dev/null; then
-      __process_msg "Database found"
-      db_booted=true
-    else
-      __process_msg "Waiting for database to start"
-      let "counter = $counter + $interval"
-      sleep $interval
-    fi
-  done
-  if [ $db_booted = false ]; then
-    __process_msg "Failed to boot database container"
-    exit 1
-  fi
+  __check_service_up $DB_IP $DB_PORT "database" "180"
```

Comment:
why are we hardcoding `database`? the env `COMPONENT` should already be there. we need to first look at the data that's already available before adding hardcoded stuff like this which degrades the system

Reply:
had hard coded name because it was being used in database specific script `installDB.sh`. I have updated it to use `$COMPONENT`